Wallet Download and Installation.md

```diff
@@ -0,0 +1,64 @@
+# Downloading and Installing Wallets to Your Devices
+
+Visit the **[Downloads](https://www.skycoin.net/downloads/)** page from the Skycoin Website to download the Skycoin Desktop Wallet and Skycoin Mobile Wallet for the device you are using:
+
+## [Desktop Wallet](Desktop_Wallet)
+
+* ### [Windows](Windows)
+* ### [MAC OS](MAC_OS)
+* ### [Linux](Linux)
+
+## [Mobile Wallet](Mobile_Wallet)
+
+* ### [IOS](IOS)
+* ### [Android](Android)
+
+## <a name="Desktop_Wallet"> </a> Desktop Wallet
+
+### <a name="Windows"> </a> Windows
+
+You can follow these steps to install the Skycoin Desktop Wallet into your Windows PC:
+
+* Clicking on the link, "64 bit" will start downloading the installer for the Wallet.
```

Comment:
Click on the download link next to "Windows Installer (Recommended)" to start downloading the wallet installer.